### ISSUE TITLE
Upcoming hard fork [view tags]

### ISSUE THREAD

**Issue by plowsof** [unyieldinggrace/monero-tx-scanner#1]
Amazing work! just wanted to make you aware about the breaking 'viewtags' in the upcoming hardfork:
https://github.com/monero-ecosystem/monero-python/issues/113

hopefully you can make sense of this half baked PR, i don't understand any of it https://github.com/monero-ecosystem/monero-python/pull/116

**Comment by unyieldinggrace** [comment]
Yeah, very aware, lol. Will have to have a crack at implementing them. But I thought packaging up this code in a stand-alone JS class was the best first step to getting it in a state where it is clean enough to be able to confidently implement future changes.
